js/ui/appIconBar.js

```diff
@@ -1004,6 +1036,23 @@ const AppIconBar = new Lang.Class({
         Main.overview.connect('showing', Lang.bind(this, this._updateActiveApp));
         Main.overview.connect('hidden', Lang.bind(this, this._updateActiveApp));
 
+
+        for (let index = 0; index < 8; index++) {
+            let fullName = 'activate-icon-' + (index + 1);
+            Main.wm.addKeybinding(fullName,
+                                  new Gio.Settings({ schema: SHELL_KEYBINDINGS_SCHEMA }),
+                                  Meta.KeyBindingFlags.NONE,
+                                  Shell.KeyBindingMode.NORMAL |
+                                  Shell.KeyBindingMode.OVERVIEW,
+                                  this._activateNthApp.bind(this, index));
```

Comment:
`Lang.bind(this, this._activateNthApp, index)` did not work. Is it terrible to use real `bind` here?